test/bin/scenario.sh

```diff
@@ -197,22 +197,36 @@ launch_vm() {
     # FIXME: variable for vcpus?
     # FIXME: variable for memory?
     # FIXME: variable for ISO
-    if timeout "${VM_BOOT_TIMEOUT}" sudo virt-install \
+
+    local -r wait=$((VM_BOOT_TIMEOUT / 60))
+    local -r install_log=$(mktemp)
+    local -r install_log_dest="${WEB_SERVER_URL}/scenario-info/${SCENARIO}/vms/${vmname}/vm_install.log"
+
+    set +e
+    sudo virt-install \
          --noautoconsole \
          --name "${full_vmname}" \
          --vcpus 2 \
          --memory 4092 \
          --disk "pool=${VM_STORAGE_POOL},size=30" \
          --network network=default,model=virtio \
-         --events on_reboot=restart \
```

Comment:
I think we need to leave this event, or attempts to reboot VMs would result in their shutdown. 
We have a few tests that reboot VMs, right?